docs/components/DocumentationSidebarRightLink.tsx

```diff
@@ -140,13 +150,15 @@ const DocumentationSidebarRightLink = React.forwardRef<HTMLAnchorElement, Sideba
           style={{ paddingLeft }}
           href={'#' + slug}
           onClick={onClick}
-          css={[
-            STYLES_LINK,
-            isNested && STYLES_LINK_HEADER,
-            isCode && STYLES_LINK_CODE,
-            isActive && STYLES_LINK_ACTIVE,
-          ]}>
-          {displayTitle}
+          css={[STYLES_LINK, isNested && STYLES_LINK_HEADER, isActive && STYLES_LINK_ACTIVE]}>
+          <span css={[STYLES_LINK_LABEL, isCode && STYLES_LINK_CODE]}>{displayTitle}</span>
+          {tags && tags.length ? (
+            <div css={css({ display: 'inline-flex' })}>
+              {tags.map((tag: string) => (
+                <Tag toc name={tag} />
```

Comment:
```suggestion
                <Tag type="toc" name={tag} />
```
For this type of prop, I prefer a "category" prop `type`, and a value, like `"toc"` or `"regular"` (which is the default), since it leaves the door open to other tag types in the future.

I did this with buttons on the website, where we used `small`, and then switched to `size="small"`, which has been nicer to work with.